tests/bot/exts/info/test_information.py

```diff
@@ -619,3 +619,34 @@ async def test_return_none_if_one_rule_number_is_invalid(self):
                 self.ctx.send.call_args,
                 unittest.mock.call(shorten(":x: Invalid rule indices: " + invalid, 75, placeholder=" ...")))
             self.assertEqual(None, final_rule_numbers)
+
+    async def test_return_correct_rule_numberstest_return_correct_rule_numbers(self):
+
+        test_cases = [
+            (("1", "2", "first"), {1, 2}),
+            (("1", "hello", "2", "second"), {1}),
+            (("second", "third", "unknown", "999"), {2, 3})
+        ]
```

Comment:
When there are multiple test cases, use the `subTest` context manager to isolate the asserts.